src/util/edit.cc

```diff
@@ -253,19 +302,22 @@ data::BinData EditEngine::getDataFromEditNode(const EditNode& edit_node,
 void EditEngine::remap(size_t pos, size_t old_size, size_t new_size) {
   // This variable can be overflowed when `old_size` > `new_size`,
   // but it will be OK when we add this difference to some address.
-  size_t offset = new_size - old_size;
+  const size_t offset = new_size - old_size;
+  // remap is O(n), so if `offset == 0` then we should use something faster
```

Comment:
It's not clear what `n` means here. Change size in bytes? Size of the mapping? (*I* know what it means, but other readers probably don't).